draft-ietf-quic-http.md

```diff
@@ -2059,26 +2064,29 @@ NOT be assigned by IANA.
 This document establishes a registry for HTTP/3 error codes. The "HTTP/3 Error
 Code" registry manages a 62-bit space.  This registry follows the QUIC registry
 policy; see {{iana-policy}}.  Permanent registrations in this registry are
-assigned using the Specification Required policy {{!RFC8126}}, except for values
-between 0x00 and 0x3f (in hexadecimal; inclusive), which are assigned using
-Standards Action or IESG Approval as defined in Section 4.9 and 4.10 of
+assigned using the Specification Required policy ({{!RFC8126}}), except for
+values between 0x00 and 0x3f (in hexadecimal; inclusive), which are assigned
+using Standards Action or IESG Approval as defined in Section 4.9 and 4.10 of
 {{!RFC8126}}.
 
 Registrations for error codes are required to include a description
 of the error code.  An expert reviewer is advised to examine new
 registrations for possible duplication with existing error codes.
 Use of existing registrations is to be encouraged, but not mandated.
 
-In addition to common fields as described in {{iana-policy}}, permanent
-registrations in this registry MUST include the following fields:
+In addition to common fields as described in {{iana-policy}}, this registry
+includes two additional fields. Permanent registrations in this registry SHOULD
+include the following field:
```

Comment:
Are you worried about the greasing values with this?

Reply:
No, just simple conflict in requirements within the text.  If Name is optional, then it can't be a MUST include it.  If it MUST be included, it's not optional.

Reply:
@martinthomson, did you have a different suggested arrangement or can I resolve this comment?

Reply:
No, I will resolve it.  It's just odd, that's all.  I would have said it was easier to require permanent registrations to have a name.